agent/functional_tests/testdata/taskdefinitions/task-shared-vol-read/task-definition.json

```diff
@@ -23,10 +23,6 @@
                 "scope": "shared",
                 "autoprovision": false,
                 "driver": "local",
-                "driverOpts": {
```

Comment:
Why has this been removed?

Reply:
I think `tmpfs` may not be a good option to test here, I saw wired things happen where the `write` happened but the file doesn't exist. I will need to investigate more about the option after this PR.